config/import-control.xml

```diff
@@ -240,6 +240,12 @@
     <allow pkg="com.puppycrawl.tools.checkstyle.xpath.iterators" local-only="true"/>
     <allow class="com.puppycrawl.tools.checkstyle.TreeWalkerAuditEvent" local-only="true"/>
 
+    <file name="Abstract.*Node" regex="true">
+      <allow pkg="com.puppycrawl.tools.checkstyle.xpath.iterators"/>
+      <!-- These are more saxon utilities, and should not have anything checkstyle specific -->
```

Comment:
>These are more saxon utilities

I am not sure what you mean by this. Rephrase or remove

Reply:
See same comment further down under `<subpackage name="iterators">`.

This was to denote classes in this package are utilities for saxon only and should not have anything Checkstyle specific, like `DetailAST`.

I duplicated the same note because this was for the same purpose.

Reply:
Now I see root of confusion.
In bellow block
```
<subpackage name="iterators">
      <!-- These are more saxon utilities, and should not have anything checkstyle specific -->
      <disallow pkg=".....
```

Comment is for whole `subpackage`. It makes sense. I don't know what is most good approach to place comments: above tag or inside tag as first line.

In new `file` tag, comment is placed not on top , so intent to describe sub tag only, not a parent tag.
Please move it at top at least.

Reply:
Done.